modules/pam_faillock/main.c

```diff
@@ -137,20 +205,23 @@ do_user(struct options *opts, const char *user)
 	const char *dir = get_tally_dir(opts);
 
 	pwd = getpwnam(user);
+	if (!pwd) {
+	    fprintf(stderr, "%s: getpwnam; no such user: %s\n", opts->progname, user);
+	    return 1;
+	}
```

Comment:
If this is a fix, it should rather go to a separate commit.

Nitpicking: why getpwnam in the diagnostics is followed by a semicolon?

Reply:
Done.